### PR TITLE
Update to datalist

### PR DESCRIPTION
**Pull request by Nordico-team**
I Tiago Rangel, brother of André Rangel, change this file to add the datalist.
It haves the Nordico list of citys, and provides a search to the citys.
PLEASE verify the code

### CODE REVIEW

**Review by siigdev — CHANGES_REQUESTED** [review]

File: src/components/landing_page/SearchBox.jsx [hunk 1]
```diff
@@ -42,14 +42,18 @@ class SearchBox extends Component {
                             </div>
                             <Form.Label htmlFor="input-city">Where?</Form.Label>
                             <InputGroup>
-                                <Form.Control
-                                    type="text"
-                                    className="mx-sm-2"
+ <Autocomplete
```
[the remark this answers is not in the document]
Comment (Nordico-team) [reply src/components/landing_page/SearchBox.jsx:45]: Okay!

File: src/components/landing_page/SearchBox.jsx [hunk 2]
```diff
@@ -42,14 +42,18 @@ class SearchBox extends Component {
...
+                                                                          className="mx-sm-2"
                                     id="input-city"
                                     aria-describedby="city-help"
                                     defaultValue={this.state.city}
                                     onChange={(event) => this.setState({ city: event.target.value })}
-                                />
+                                    
+                                    
+        onSelect={handleOnSelect} 
+        onSearch={handleOnSearch}
```
[the remark this answers is not in the document]
Comment (Nordico-team) [reply src/components/landing_page/SearchBox.jsx:54]: Sorry, I don't delete them.
I was using a code demo.

[the remark this answers is not in the document]
Comment (Nordico-team) [reply src/components/landing_page/SearchBox.jsx:54]: I well delete it now!

Comment (Nordico-team) [inline src/components/landing_page/SearchBox.jsx:54]: For deliting

[the remark this answers is not in the document]
Comment (Nordico-team) [reply src/components/landing_page/SearchBox.jsx]: Deleted!

File: src/components/landing_page/SearchBox.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Nordico-team) [reply src/components/landing_page/SearchBox.jsx:45]: Sorry, but how can I correct it?
(By Tiago Rangel)

[the remark this answers is not in the document]
Comment (Nordico-team) [reply src/components/landing_page/SearchBox.jsx:45]: How, there is! On the other code, you don't close the > or the < on the first tag.
Sorry...

[the remark this answers is not in the document]
Comment (Nordico-team) [reply src/components/landing_page/SearchBox.jsx:45]: I well try to correct it!

File: src/styles/main.scss [hunk 3]
```diff
@@ -67,6 +67,7 @@ nav {
 .nav-link {
   text-decoration: none !important;
   color: white;
+  cursor:pointer;
```
Comment (Nordico-team) [inline src/styles/main.scss:70]: I've just added the cursor:pointer so the user know's that it is a place that he can click.

**Comment by Nordico-team** [comment]
Closed
Too many time to respond

**Pull request closed without merging**